### ISSUE TITLE
Searches in the wrong folder

### ISSUE THREAD

**Issue by To1ne** [pjaspers/stringer#4]
`> rake localize`
>Generating en.lproj
>rake aborted!
>No Localisations found at /Users/toon/.gem/ruby/2.2.0/gems/stringer-0.0.1/lib/stringer/stringer/en.lproj/Localizable.strings

I had to write my rake task like this:
```ruby
desc "Run genstrings to update the Localizable.strings files"
task localize: :env do
  folder = File.join(File.dirname(__FILE__), @appname)
  %w(en).each do |locale|
    Stringer.run(locale, files_folder: folder, lproj_parent: folder)
  end
end
```

Or am I doing something wrong here?

**Comment by pjaspers** [comment]
No, you're probably right (although since you're using ruby 2.2, you could use `__DIR__` instead of `File.dirname(__FILE__)`).

Just setting the `lproj_parent` should work as well.
